### PR TITLE
Remove useless comment

### PR DESCRIPTION
**Pull request by little-jade**

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Please, do not use parseInt with a number, and don't use octal literals (the literal `015` itself represents the number 13). The current text is fully correct.

**Comment by little-jade** [comment]
@yin1999 
ref: https://github.com/mdn/translated-content/pull/18252

**Comment by Josh-Cena** [comment]
The zh content is missing quotes; that should be fixed instead

**Comment by little-jade** [comment]
I know what you mean. But I guess the original intent of this comment is to remind the user not to forget to put quotation marks, otherwise there will be unexpected results. But the original author may have misspelled `parseInt(015, 10)` as `parseInt(015, 8)`. If we don't want to show bad writing in the comment, we should remove the comment instead of changing it to `parseInt('015', 8)` , which will confuse the intent of the comment.

**Comment by Josh-Cena** [comment]
I think you have a point. Let's remove this comment.

File: files/en-us/web/javascript/reference/global_objects/parseint/index.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ The following examples all return `15`:
 parseInt("0xF", 16);
 parseInt("F", 16);
 parseInt("17", 8);
-parseInt("015", 10); // but `parseInt('015', 8)` will return 13
+parseInt("015", 10); // but `parseInt(015, 10)` will return 13
```
Comment (jasonren0403) [inline files/en-us/web/javascript/reference/global_objects/parseint/index.md:81]:
```suggestion
parseInt("015", 10);
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**